tests/lib/dbus.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+DISPLAY=:0
+
+# shellcheck source=tests/lib/pkgdb.sh
+. "$TESTSLIB/pkgdb.sh"
+
+init_dbus_env(){
+    export $(cat dbus.env | xargs)
+}
+
+start_dbus_loop_unit(){
+    local executable="$1"
+
+    distro_install_package dbus-x11
+
+    dbus-launch > dbus.env
+    init_dbus_env
```

Comment:
Ok, xargs removed

I'm not sure it's a good idea to force the clients of this code to know there's a `dbus.env` file, all the handling of that file is done inside `tests/lib/dbus.sh` and IMO it's better to hide this implementation detail behind the function name, WDYT?